api-gateway/src/main/kotlin/com/saveourtool/save/gateway/security/WebSecurityConfig.kt

```diff
@@ -177,18 +185,25 @@ fun passwordEncoder(): PasswordEncoder = PasswordEncoderFactories.createDelegati
  * @param authorizationContext
  * @return [Mono] of [AuthorizationDecision]
  */
-private fun defaultAuthorizationDecision(
+private fun authorizationManagerAuthorizationDecision(
```

Comment:
can you please move it below method `defaultAuthorizationDecision`, where it is called ?

Reply:
Moved all dependant methods below `defaultAuthorizationDecision` and `userStatusBasedAuthorizationDecision`